lib/Dialect/FIRRTL/Transforms/ExpandWhens.cpp

```diff
@@ -132,12 +132,19 @@ class LastConnectResolver : public FIRRTLVisitor<ConcreteT> {
   using FIRRTLVisitor<ConcreteT>::visitDecl;
   using FIRRTLVisitor<ConcreteT>::visitStmt;
 
-  /// Records a connection to a destination in the current scope. This will
-  /// delete a previous connection to a destination if there was one. Returns
-  /// true if an old connect was erased.
-  bool setLastConnect(FieldRef dest, Operation *connection) {
+  /// Records a connection to a destination in the current scope. For
+  /// last-connect connections, this will delete a previous connection to a
```

Comment:
For singlely connected elements?  Wording seems confusing.  Last-connect in firrtl specifically refers to what happens when there are multiple connects.

Reply:
Oh, I'll work on clarifying the two supported types of connects--your suggestion was spot-on so I renamed the method on FConnectLike but you're right it's not evident or even necessary that not being "static single connect" means "last-connect".  (and I'll be sure to address your comment as phrased/the confusion :+1:)